skaffold/src/test/kotlin/com/google/container/tools/skaffold/run/SkaffoldCommandLineStateTest.kt

```diff
@@ -141,4 +143,32 @@ class SkaffoldCommandLineStateTest {
             )
         )
     }
+
+    @Test
+    fun `A run error is thrown if skaffold isn't in the system PATH`() {
+        every { SkaffoldExecutorService.instance.isSkaffoldAvailable() } answers { false }
+
+        skaffoldCommandLineState = SkaffoldCommandLineState(
+            mockExecutionEnvironment,
+            SkaffoldExecutorSettings.ExecutionMode.DEV
+        )
+
+        expectThrows(
+            ExecutionException::class,
+            ThrowableRunnable { skaffoldCommandLineState.startProcess() })
+    }
+
+    @Test
+    fun `A run error isn't thrown if skaffold is in the system PATH`() {
```

Comment:
I’d remove all the single quotes from these test names. It was causing weird behavior when running the tests from the IDEAs JUnit runner.